### ISSUE TITLE
Please add support for generic types restrains

### ISSUE THREAD

**Issue by MZhoume** [k--kato/vscode-docomment#35]
Hello Keisuke,

Sweet plugin! But please add support for generic types when there are restrains applied. 

E.g.:
```C#
        /// <summary>
        /// Cannot auto generate fields for params and return type
        /// </summary>
        public string Test<T>(T obj) where T : class
        {
            // do something...
        }
```

**Comment by k--kato** [comment]
Thanks. I fixed it and released v0.0.14, check it out.

**Comment by MZhoume** [comment]
@k--kato Works great, thanks ;)
